documentation/reference/document-json-format.html

```diff
@@ -941,6 +944,25 @@ <h3 id="tensor-add">Add tensor cells</h3>
 In this example, cell <code>{"x":"b","y":"2"}</code> will implicitly be set will value 0.0.
 </p>
 
+<p>
+Prefer the <i>block short form</i> for mixed tensors instead. This also avoids the problem where cells with indexed dimensions are not specified:
```

Comment:
&lt;i&gt; is deprecated in html5 afaik, &lt;em&gt; is a good alternative

Reply:
Fixed